src/app/services/user.js

```diff
@@ -202,6 +210,36 @@ module.exports = (Model, App) => {
       throw new Error('User not found');
     }
 
+    const [mailLimit] = await Model.mailLimit.findOrCreate({
+      where: {
+        userId: user.id,
+        mailType: 'deactivate_user',
+      },
+      default: {
+        attemptsCount: 0,
+        attemptsLimit: 10,
+      },
+    });
+
+    if (mailLimit.attemptsCount >= mailLimit.attemptsLimit) {
```

Comment:
See https://github.com/internxt/drive-server/pull/364 for reference, same issue is there.